Company.py

```diff
@@ -0,0 +1,14 @@
+class Company:
+    def __init__(self, name, address, employees=[]):
+        self.name = name
+        self.address = address
+        self.employees = employees
+
+    def add_employee (self, employee):
+        try:
+            if employee.salary>5000 and employee.salary<20000:
+                self.employees.append(employee)
+            else:
+                raise ValueError
+        except ValueError:
+            print('Could not add'+ (self.employees) +' new employee; employee salary is less than 5000 or greater than 20000')
```

Comment:
This line throws `TypeError: can only concatenate str (not "list") to str`. 
In any case, `self.employees` would print every employee in the company, and here we'd like to print just the employee that cannot be added. Maybe it's a typo, and there is an extra "s" in `self.employees`?

Also, when concatenating strings this way, no parentheses are needed around the variable.

Reply:
Deleted, program is just printing 'Could not add new employee; employee salary is less than 5000 or greater than 20000'

Reply:
I don't see the changed code, forgot to push? :)

In any case, printing info about the employee that caused the exception would be preferable.

Reply:
Program is working, I added printing employee that caused exception.